client/styles/upe/index.js

```diff
@@ -345,19 +382,42 @@ export const getAppearance = ( isBlocksCheckout = false ) => {
 		color: selectedTabRules.color,
 	};
 
+	const paragraphRules = getFieldStyles(
+		selectors.upeThemeTextSelectors,
+		'.Text'
+	);
+
+	const darkParagraphRules = getFieldStyles( selectors.hiddenInput, '.Text' );
+
+	const globalRules = {
+		colorBackground: backgroundColor,
+		colorText: isColorLight( backgroundColor )
+			? inputRules.color
+			: paragraphRules.color,
+		fontFamily: paragraphRules.fontFamily,
+		fontSizeBase: paragraphRules.fontSize,
+	};
+
 	const appearance = {
+		variables: globalRules,
+		theme: isColorLight( backgroundColor ) ? 'stripe' : 'night',
 		rules: {
 			'.Input': inputRules,
 			'.Input:focus': inputFocusRules,
 			'.Input--invalid': inputInvalidRules,
+			'.Block': blockRules,
 			'.Label': labelRules,
 			'.Tab': tabRules,
 			'.Tab:hover': tabHoverRules,
 			'.Tab--selected': selectedTabRules,
 			'.TabIcon:hover': tabIconHoverRules,
 			'.TabIcon--selected': selectedTabIconRules,
-			'.Text': labelRules,
-			'.Text--redirect': labelRules,
+			'.Text': isColorLight( backgroundColor )
+				? darkParagraphRules
+				: paragraphRules,
+			'.Text--redirect': isColorLight( backgroundColor )
+				? darkParagraphRules
```

Comment:
`darkParagraphRules` are obtained from the hidden input field. When using a dark palette, we use input text color for non-input text. This is a bit hacky but needed because, in shortcode checkout, there's always a light overlay. So, if we use paragraph rules, we will have light text on a light background.

Reply:
I see this for block checkout, 2025 theme. Is this expected?

<img width="835" alt="Screenshot 2025-01-29 at 4 08 38 PM" src="https://github.com/user-attachments/assets/dbbeda86-04d6-4dfa-bb1b-441e7b4b93a3" />

Reply:
@annemirasol That doesn't look right. This is what I get 
![CleanShot 2025-01-29 at 19 37 20](https://github.com/user-attachments/assets/99b2e468-a085-4524-a51c-187302b16b4c)

Can you double-check to make sure it's not using the cached styles?